README.md

```diff
@@ -464,12 +464,18 @@ With this you can customize the classes applied to various tree elements
 (`treeClass`, `emptyTreeClass`, `hiddenClass`, `nodesClass`, `handleClass`,
 `placeholderClass`, `dragClass`).
 
-In addition, you can modify whether nodes are collapsed by default
-(`defaultCollapsed`: default false). For example:
+In addition, you can modify whether or not nodes are collapsed by default
+(`defaultCollapsed`: default false).
```

Comment:
I suggest making `true` the default value. We have a CHANGELOG that clearly lists what the impact is of upgrading.

@stefancodes @jacobscarter what do you guys think about this?